doc/examples/mzml-mzqc-example.mzML

```diff
@@ -325,9 +325,7 @@
 		</dataProcessing>
 	</dataProcessingList>
 	<run id="ru_0" defaultInstrumentConfigurationRef="ic_0" sampleRef="sa_0" startTimeStamp="2009-08-09T22:32:31" defaultSourceFileRef="sf_ru_0">
-		<cvParam cvRef="QC" accession="QC:4000179" name="Charged spectra ratio +1 over +2" value="0" />
-		<cvParam cvRef="QC" accession="QC:4000180" name="Charged spectra ratio +3 over +2" value="0.5454545454545454" />
-		<cvParam cvRef="QC" accession="QC:4000181" name="Charged spectra ratio +4 over +2" value="0.012987012987012988" />
+		<cvParam cvRef="QC" accession="MS:4000063" name="MS2 known precursor charges fractions" value="{'MS:1000041': [1,2,3,4], 'UO:0000191': [0,17369,10733,2256]}" />
```

Comment:
I think it needs to be changed a little bit. Are these values now the absolute number of spectra with specific charge states, rather than the fractions of spectra?

Reply:
The `cvRef` also needs to be changed to `MS`.

Reply:
d'oh! sry